src/main/java/yahav/photomosaic/ImagePixels.java

```diff
@@ -0,0 +1,72 @@
+package yahav.photomosaic;
+
+import javafx.geometry.Rectangle2D;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.scene.image.PixelFormat;
+import javafx.scene.image.PixelReader;
+import javafx.scene.paint.Color;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ImagePixels{
+
+    ArrayList<Rectangle2D> squareInput = new ArrayList<>(); //array of input squares to compare to source images
+    public ImagePixels() throws IOException {
+        FileInputStream input = new FileInputStream("src/main/resources/google.png");
+        Image image = new Image(input);
+        ImageView imageView = new ImageView();
+        PixelReader pixelReader = image.getPixelReader();
+    }
+
+
+    public Color getImagePixels(Image image) {
+        int width = (int) image.getWidth();
+        int height = (int) image.getHeight();
+        int squares = 50;
+        int xInc = width / squares;
+        int yInc = height / squares;
+        double redPixels = 0;
+        double greenPixels = 0;
+        double bluePixels = 0;
+        double numPixels = 0;
+        PixelReader pixelReader = image.getPixelReader();
+        for (int squareX = 0; squareX < height; squareX += xInc) {
+            for (int squareY = 0; squareY < width; squareY += yInc) {
```

Comment:
This method is returning the average pixel color of the whole image.
You are "breaking" the image up into 50x50 squares, but you are only returning one color so what's the point of breaking it into these squares?